Tests/HubTests/HubTests.swift

```diff
@@ -118,4 +118,14 @@ class HubTests: XCTestCase {
         let vocab_dict = config.dictionary["vocab"] as! [String: Int]
         XCTAssertNotEqual(vocab_dict.count, 2)
     }
+
+    func testConfigTokenValueDifferentOrder() {
```

Comment:
Thanks for the test! Could we please add another one that loads a tokenizer such as `facebook/bart-large-mnli`? We don't need anything fancy: just show that the tokenizer loads and that it can encode a string.